js/src/reports/compare-table-card.js

```diff
@@ -115,11 +111,7 @@ const CompareTableCard = ( {
 		metrics.map( ( metric ) => {
 			const value = row.subtotals[ metric.key ];
 			return {
-				display: metric.formatFn(
-					value,
-					storeCurrencyConfig,
-					adsCurrencyConfig
-				),
+				display: metric.formatFn( value ),
```

Comment:
❓ Same as before. Not clear for me what's this `formatFn` coming from. If it's the previous `formatAmount` why it doesn't have `(value, true)` as params?